rest/admin_api.go

```diff
@@ -370,6 +370,10 @@ func (h *handler) handleGetRawDoc() error {
 
 	doc, err := h.db.GetDocument(docid, db.DocUnmarshalSync)
 
+	if err != nil {
+		return err
```

Comment:
Might as well replace the final `return err` with `return nil` to be clear it's handled elsewhere now.